docs/Community/Governance.md

```diff
@@ -0,0 +1,137 @@
+users.
+
+Community Governance Structure
+------------------------------
+
+*Note(Aeva) section added*
+
+The Open Enclave SDK community uses a
+[concensus-seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+approach to governance at all levels. To support a growing project, we have
+adopted a loose heirarchy wherein domain-specific decisions are handled within
+each domain, or Special Interest Group (**SIG**), cross-domain work is enabled
+through Working Groups (**WG**), and a Community Governance Committee (**CGC**)
+meets to review the overall structure, ensuring the health of the project.
+
+Each group (SIGs, WGs, and CGC) maintain documentation in this repository.
+Creating a new SIG or WG requires the approval of the CGC. SIGs are generally
+responsible for code, whereas a WG is not.
+
+SIG leaders are responsible for the code within their domain, and should be
+considered authoritative for that subsection of the project's code base insofar
+as code reviews and architectural discussions. When a change affects code owned
+by multiple SIGs, it should be coordinated with the respective SIG leads.
+Cross-SIG efforts can be coordinated by establishing a Working Group (WG), which
+is given a charter and which reports progress back to the relevant SIG leaders
+or the CGC.
+
+If there is conflict, either within or between SIGs, or between any members of
+the community, wherein it is desirable, the issue can be raised to the CGC.
+
+Technical Discussions
+---------------------
+
+*Note(Aeva) section added*
+
+Technical discussions generally happen in one of these places:
+
+* [on the GitHub project](https://github.com/openenclave/openenclave), either in an Issue or a Pull Request
+* in public SIG or WG meetings, which are typically held on Zoom (public calendar TBD)(TODO)
+* on our mailing list or chat server (details TBD) (TODO)
+
+While we strive for openness and transparency, it is natural that some
+discussions will also happen in person (eg, in the hallway track) or other
+private channels. The outcome of such discussions should be considered
+non-binding until the information is shared through an open medium and relevant
+stake-holders have opportunity to contribute to the discussion.
+
+All SIG and WG leaders are responsible for creating and publishing minutes from
+the meetings they hold, ensuring that the meetings are open and inclusive, and
+that decisions they make are fair and transparent.
+
+If a private discussion pertains to an open issue or PR, and results in a
+decision, that item should be updated and reference given to the conversation.
+
+Issue/PR Review Process
+-----------------------
+
+*Note(Aeva) section updated*
+
+The community strives to triage and respond to all incoming issues within one
+week. If you haven't heard anything by then, or an issue or PR is not getting
+any response within a resonable time after it is triaged, please feel free to
+remind us with a ping on the thread.
```

Comment:
Change "thread" to "issue or PR"

Reply:
Is this critique valuable?

Isn't a discussion on either a PR or an issue a "thread"?